src/main/scala/org/bytedeco/sbt/javacpp/JavaCppPlugin.scala

```diff
@@ -0,0 +1,140 @@
+package org.bytedeco.sbt.javacpp
+
+import java.net.URLClassLoader
+import java.util.Properties
+
+import sbt.Keys._
+import sbt._
+
+import scala.language.{ postfixOps, reflectiveCalls }
+import scala.util.Try
+
+object JavaCppPlugin extends AutoPlugin {
```

Comment:
Why did you create a new file?

Reply:
This is not required. But when there is another `Plugin` object, which I'd like to enable/disable, I would not have to specify the package.